lib/matplotlib/backend_bases.py

```diff
@@ -2791,6 +2791,12 @@ def mouse_move(self, event):
             except (ValueError, OverflowError):
                 pass
             else:
+                ax = event.inaxes
+                artists = ax.artists + ax.images + ax.lines
```

Comment:
Should also look at `ax.collections` (that is where things like scatter live) and maybe `ax.patches`